parsec/core/gui/tr/parsec_en.po

```diff
@@ -3105,3 +3105,48 @@ msgstr "Hide deactivated workspaces"
 #~ msgstr ""
 #~ "A recovery file for this device "
 #~ "already exists in the selected location."
+
+msgid "ACTION_ENABLE_BLOCK_REMANENCE"
+msgstr "Keep all workspace data locally"
+
+msgid "ACTION_DISABLE_BLOCK_REMANENCE"
+msgstr "No longer keep all workspace data locally"
+
+msgid "ACTION_WORKSPACE_MANAGE_REMANENCE"
+msgstr "Manage data availability"
+
+msgid "TEXT_REMANENCE_WAITING"
+msgstr "Please wait..."
+
+msgid "TEXT_BLOCK_REMANENCE_ENABLED"
+msgstr "The data in this workspace is currently kept locally for offline access."
+
+msgid "TEXT_BLOCK_REMANENCE_DISABLED_cache_size"
+msgstr "Only a maximum of {cache_size} of data is kept locally for this workspace."
+
+msgid "TEXT_WORKSPACE_SIZE_local_size_total_size"
+msgstr "{local_size} / {total_size} of data are available locally."
+
+msgid "TEXT_BLOCK_REMANENCE_WARNING_remote_size"
+msgstr "Keeping data locally for offline access would require to download {remote_size} of data."
+
+msgid "TEXT_GET_REMANENCE_STATUS_FAILED"
+msgstr "Workspace information is not available at the moment."
+
+msgid "TEXT_DISABLE_BLOCK_REMANENCE_FAILED"
+msgstr "An error occured while changing the data management policy."
+
+msgid "TEXT_ENABLE_BLOCK_REMANENCE_FAILED"
+msgstr "An error occured while changing the data management policy."
+
+msgid "TEXT_REMANENCE_DIALOG_TITLE_workspace_name"
+msgstr "Data management for workspace {workspace_name}"
+
+msgid "TEXT_ENABLE_BLOCK_REMANENCE_SUCCESS"
+msgstr "The data in this workspace will be now kept locally for offline access."
```

Comment:
`will not be kept`

Reply:
8cb29260f7c9dc35c0b347b22db0cbf02202ac5e